Magento2/Tests/Whitespace/MultipleEmptyLinesUnitTest.inc.fixed

```diff
@@ -0,0 +1,24 @@
+<?php
+class Foo
+{
+    private $foo;
+    private $bar;
+
+    private $baz;
+    public function test()
+    {
+        $someValue = 1;
+
+        $someValue = 2;
+
+        $someValue = 3;
+
+        $someValue = 40;
+
+        $someValue = 'newlines and spaces and tabs';
+
+$someValue = 'no indentation';
```

Comment:
it doesn't look ok. are you sure it's fine?

Reply:
@ihor-sviziev what do you expect to see here? This sniff deals with "multiple empty lines".

Reply:
I thought having spaces would be good. But in terms of this rule, it does make sense. Thx for the explanation